similarity/src/test/java/edu/illinois/cs/cogcomp/mrcs/ParaphraseTest.java

```diff
@@ -0,0 +1,77 @@
+package edu.illinois.cs.cogcomp.mrcs;
+
+import edu.illinois.cs.cogcomp.llm.comparators.LlmStringComparator;
+import org.apache.commons.math3.stat.correlation.PearsonsCorrelation;
+import java.io.BufferedReader;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.HashMap;
+
+/**
+ * Created by sling3 on 4/12/17.
+ */
+public class ParaphraseTest {
```

Comment:
is there a reason this is not a Unit test? IF so, document reason in class Javadoc